From 3e3d255f74cfec3c26156d0acf94b5822669b3d1 Mon Sep 17 00:00:00 2001
Message-ID: <3e3d255f74cfec3c26156d0acf94b5822669b3d1.1699403992.git.sharlatanus@gmail.com>
In-Reply-To: <0117df90b1ef43a67d4fa86e5b58e60f7c0a52e5.1699403992.git.sharlatanus@gmail.com>
References: <0117df90b1ef43a67d4fa86e5b58e60f7c0a52e5.1699403992.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Sat, 28 Oct 2023 03:26:05 +0100
Subject: [PATCH 11/67] gnu: go-github-com-gorilla-handlers: Remove input
 labels.

* gnu/packages/golang-web.scm (go-github-com-gorilla-handlers): Remove labels.

Change-Id: I955408970a64d03823327912fdafcac5d7fbda93
---
 gnu/packages/golang-web.scm | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/gnu/packages/golang-web.scm b/gnu/packages/golang-web.scm
index 0480f648eb..3a3c5f9621 100644
--- a/gnu/packages/golang-web.scm
+++ b/gnu/packages/golang-web.scm
@@ -154,11 +154,11 @@ (define-public go-github-com-gorilla-handlers
        (sha256
         (base32 "15gycdz9lkjnsvvichsbdf25vf6pi1sfn41khhz53iqf300l0w0s"))))
     (build-system go-build-system)
-    (propagated-inputs
-     `(("github.com/felixge/httpsnoop" ,go-github-com-felixge-httpsnoop)))
     (arguments
      '(#:tests? #f                      ; Tries to download from the internet
        #:import-path "github.com/gorilla/handlers"))
+    (propagated-inputs
+     (list go-github-com-felixge-httpsnoop))
     (home-page "https://github.com/gorilla/handlers")
     (synopsis "Middleware for Go HTTP services and web applications")
     (description "A collection of useful middleware for Go HTTP services and
-- 
2.41.0

